JSDemos/Demos/TreeList/RowEditing/description.md

```diff
@@ -1 +1,16 @@
-This example demonstrates editing in _&laquo;row&raquo;_ mode. Each row is&nbsp;supplied with an _&laquo;Edit&raquo;_ link, a&nbsp;click on&nbsp;which switches the row into the editing state. Editors vary from one cell to&nbsp;another: they may be&nbsp;calendars, select boxes, simple text boxes, etc. Note that you can also add or&nbsp;delete a&nbsp;row using the _&laquo;Add a&nbsp;row&raquo;_ button or&nbsp;the _&laquo;Delete&raquo;_ link, respectively.
\ No newline at end of file
+The TreeList allows users to edit data in multiple modes. This demo shows the *"row"* edit mode. To edit any row, click Edit in it. Only one row can be in the edit state at a time.
+
+To enable row edit mode, set the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) property to *"row"* and enable the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties to allow edit operations.
+
+Buttons for allowed edit operations are displayed in an edit column. To customize it programmatically, declare a column of the `buttons` [type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#type) and specify available buttons. In this demo, the `buttons` column is used to hide an Add button from each row. See the following help topic for more details on edit column customization: [Customize the Edit Column](/Documentation/Guide/UI_Components/TreeList/Columns/Column_Types/Command_Columns/#Customize_the_Edit_Column).
+
+
+This demo also illustrates the following event handlers:
+
+- [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow)        
+Populates cells of a new row with default values. In this demo, **onInitNewRow** sets John Heart as Head for new rows.
```

Comment:
`sets John Heart as Head for new rows` Explain

Reply:
When the button to add a new row is clicked, the new row editor's Head field is prepopulated:
![image](https://user-images.githubusercontent.com/66619750/114580757-d00d9080-9c87-11eb-84ca-dbd4009d0f1c.png)

Reply:
```suggestion
Populates cells of a new row with default values. In this demo, **onInitNewRow** sets John Heart as the initial Head for new rows.
```